src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -17,60 +17,52 @@ const propTypes = {
     ...windowDimensionsPropTypes,
 };
 
-class BaseSidebarScreen extends Component {
-    constructor(props) {
-        super(props);
+/**
+ * Method called when avatar is clicked
```

Comment:
NAB - Technically this isn't a method anymore, right? Since it's not inside the component? I might be wrong about that.